src/ontology/mondo-edit.obo

```diff
@@ -435350,21 +435336,35 @@ is_a: MONDO:0019250 {source="Orphanet:79169"} ! inborn disorder of biogenic amin
 [Term]
 id: MONDO:0019220
 name: inborn disorder of cobalamin metabolism and transport
+def: "An inherited metabolic disease affecting cobalamin (vitamin B12) intestinal absorption, transport in the blood, uptake by peripheral cells or cellular metabolism." [PMID:35337626]
 subset: clingen {source="MONDO:CLINGEN"}
 subset: disease_grouping
 subset: gard_rare {source="GARD:18951", source="MONDO:GARD"}
+subset: gard_rare {source="MONDO:GARD"}
 subset: ordo_group_of_disorders {source="Orphanet:79171"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "cobalamin deficiency" BROAD [DOID:0050731]
 synonym: "disorder of cobalamin metabolism and transport" RELATED [Orphanet:79171]
+synonym: "hypocobalaminemia" BROAD [DOID:0050731]
 synonym: "inborn disorder of cobalamin metabolism and transport" EXACT CLINGEN_LABEL []
+synonym: "inborn error of cobalamin metabolic process" EXACT [MONDO:patterns/inborn_metabolic]
+synonym: "inborn vitamin B12 deficiency (disease)" EXACT [https://orcid.org/0000-0002-6601-2165]
+synonym: "rare inborn error of cobalamin metabolic process" EXACT [MONDO:patterns/inborn_metabolic]
+xref: DOID:0050731 {source="MONDO:equivalentTo"}
 xref: GARD:18951 {source="MONDO:GARD"}
 xref: icd11.foundation:936546617 {source="Orphanet:79171", source="MONDO:equivalentTo", source="https://orcid.org/0000-0001-5208-3432"}
 xref: MEDGEN:1826150 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:79171 {source="MONDO:equivalentTo"}
 xref: UMLS:C5681844 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1826150"}
+is_a: MONDO:0005528 {source="DOID:0050731", source="HP:0100502", source="MONDO:Entailed", source="MONDO:Redundant"} ! inborn vitamin metabolic disorder
 is_a: MONDO:0017758 {source="Orphanet:79171"} ! disorder of vitamin and non-protein cofactor absorption and transport
+intersection_of: MONDO:0019052 ! inborn errors of metabolism
+intersection_of: disease_has_basis_in_disruption_of GO:0009235 ! cobalamin metabolic process
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0019220 {source="MONDO:CLINGEN"}
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7353" xsd:anyURI
+property_value: IAO:0000589 "inborn vitamin B12 deficiency (disease)" xsd:string
```

Comment:
Do you know where this came from?

Reply:
I think it is given when the label is the same as another label in obo (e.g. as a phenotype). 
I think we can remove this annotation and make sure that the name is a synonym

Reply:
removed this annotation and made sure that the name "inborn vitamin B12 deficiency (disease)" is listed as a synonym.